src/System.Net.Http/src/System/Net/Http/Headers/ProductInfoHeaderValue.cs

```diff
@@ -155,7 +155,7 @@ internal static int GetProductInfoLength(string input, int startIndex, out Produ
             }
             else
             {
-                // Trailing whitespaces are removed by GetProductLength().
+                // Trailing whitespace are removed by GetProductLength().
```

Comment:
Mass noun ("some whitespace") or adjective used abbreviated as a noun ("whitespace characters" -> "whitespace"). Either way it probably shouldn't be given a trailing *s*.